packages/lesswrong/components/posts/TableOfContents/ToCColumn.tsx

```diff
@@ -128,14 +150,15 @@ export const ToCColumn = ({tableOfContents, header, welcomeBox, children, classe
       classes.root,
       {
         [classes.tocActivated]: !!tableOfContents || !!welcomeBox,
+        [classes.tocActivatedCenterEarly]: centerEarly,
         [classes.sideCommentsActive]: sideCommentsActive,
       }
     )}>
       <div className={classes.header}>
         {header}
       </div>
       {tableOfContents && <div className={classes.toc}>
-        <div className={classes.stickyBlockScroller}>
+        <div className={classNames(classes.stickyBlockScroller, classes.stickyBlockScrollerCenterEarly)}>
```

Comment:
I believe you mean to be conditionally applying this class and the fact that you're not is the cause of this bug:

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/10352319/217413017-e5742e13-1fa8-466d-825f-2f86ba95cdd5.png">